examples/common_vmseries/variables.tf

```diff
@@ -891,6 +891,12 @@ variable "test_infrastructure" {
                                   [VNET module documentation](../../modules/vnet/README.md#network_security_groups).
     - `route_tables`            - (`map`, optional) map of Route Tables to create, for details see
                                   [VNET module documentation](../../modules/vnet/README.md#route_tables).
+  - `local_peer_config`       - (`map`, optional) a map that contains local peer configuration parameters. This value allows to 
```

Comment:
You also need to indent it more as it's nested under `vnets` object (should be on par with e.g. the `route_tables` property above).

Reply:
Copy, past issue, sorted